
void foo() throws MyException {}
void bar() throws Throwable {}
